test/testdata/packager/import-autocorrect/__package.1.rbedited 

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+# typed: strict
+# enable-packager: true
+
+class Package < PackageSpec
+  import Dep
+end
```

Comment:
When I add a line to the end of this file, like "random string", the test still passes. I do not think this test is actually running.